terraform/application/config/migration.tfvars.json

```diff
@@ -7,5 +7,9 @@
     "redis_cache_capacity": 0,
     "redis_cache_sku_name": "Basic",
     "enable_logit": true,
-    "worker_replicas": 5
+    "worker_replicas": 5,
+    "webapp_replicas": 4,
```

Comment:
Ditto as with the ECF PR; not sure we need to scale the webapps

Reply:
Is there any point in having more than 1 webapp?